llama-index-integrations/llms/llama-index-llms-deepinfra/pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.llms.deepinfra"
+
+[tool.llamahub.class_authors]
+DeepInfraLLM = "llama-index"
```

Comment:
feel free to change `llama-index`, which is more of a placeholder, to your own Github username